src/api.js

```diff
@@ -19,6 +19,20 @@ export const MOCK_DUOTONE_URLS = {
 export const MOCK_GROUP = {
 	id: 1234,
 	urlname: 'fake-hq-faff',
+	organizer: {
```

Comment:
- I had consider referencing `MOCK_MEMBER` here but the `organizer` value I'm seeing at https://www.dev.meetup.com/meetup_api/docs/:urlname#get looks a touch different
- Also considered making a `MOCK_ORGANIZER` to reference in `MOCK_GROUP`
- Do we always prefer place kittens for photos?